code/game/gamemodes/revolution/revolution.dm

```diff
@@ -423,22 +249,21 @@
 	for(var/datum/mind/M in SSticker.mode:revolutionaries)
 		if(M.current && M.current.stat != DEAD)
 			revcount++
+
+	var/list/real_command_positions = GLOB.command_positions.Copy() - "Nanotrasen Representative"
 	for(var/thing in GLOB.human_list)
 		var/mob/living/carbon/human/player = thing
```

Comment:
Unless there are nulls
```suggestion
	for(var/mob/living/carbon/human/player as anything in GLOB.human_list)
```

Reply:
Applied without the `as anything` for safety